master/getting-started/bare-metal/bare-metal.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Using Calico to Secure Host Interfaces
+redirect_from: latest/getting-started/bare-metal/bare-metal
```

Comment:
I do not think that we want this redirect line here, in the master folder. This redirect would only be included in MD files of the version that is *actually* the latest, which, at present is 2.5. @ozdanborne is out, but can confirm perhaps when he returns. Still, I believe that you can look in the source files to confirm that this is the case.

Reply:
Yes. You are right.